### DISCUSSION TITLE
Speed is very fast but the accuracy of the task is pretty low? Why?

### DISCUSSION THREAD

**Discussion by AmazeQiu** [vllm-project/vllm#768] [Q&A]
We finetuned llama2-13B to meet our text2sql task requirements, it worked OK with hf transformers in offline testing: the accuracy is 0.72, latency is about 6.5s
But when I deployed the model with vllm, the accuracy becomes pretty low: accuracy falls to 0.57, latency is about 2.6s
I use hf with this config:
**temperature=0, top_p=1.0 ,top_k=40 ,num_beams=4, max_new_tokens=100**
then use vllm with this config :
**max_tokens = 100, use_beam_search = True, best_of = 4, temperature = 0 ,top_p = 1.0, top_k = -1**
The speed is fast but the accuracy is pretty lower than that using hf. T _ T
I even change the source code to use "model.half().cuda()" to make it same with the way I use hf in my code.
WHY? T _ T

**Comment by admangan** [comment]
I noticed the same on a finetuned llama2 13B. Worked really well with HF transformers but when I loaded into vllm (which took a while) the accuracy was way down. Especially stop word usage, which is odd. In the HF inference the stopwords used in teh finetuning were always present (even if generation was incorrect) but in the vllm they were not.
